jsonpickle/pickler.py

```diff
@@ -584,4 +649,4 @@ def _wrap_string_slot(string):
     """
     if isinstance(string, string_types):
         return (string,)
-    return string
+    return string
```

Comment:
New newline at end of file?  What editor are you using?  Please adjust your settings.

Reply:
I use VSCode. Not sure why that happened, I don't have it set to do that as far as I am aware.